src/main/scala/mesosphere/marathon/core/election/CuratorElectionStream.scala

```diff
@@ -220,7 +265,7 @@ object CuratorElectionStream extends StrictLogging {
       override def getAclForPath(path: String): util.List[ACL] = defaultAcl
     }
 
-    val retryPolicy = new ExponentialBackoffRetry(1.second.toMillis.toInt, 10)
+    val retryPolicy = new ExponentialBackoffRetry(1.second.toMillis.toInt, 3)
```

Comment:
hmmm ... so there is connection retry on curator level as well as our own retry on the watch level?

Reply:
^ I will put this as a comment TODO